contributors/guide/contributor-cheatsheet/README.md

```diff
@@ -43,7 +43,7 @@ better.
   Project.
 - [Developer Guide] - Guide to contributing code directly to the Kubernetes
   Project.
-
+- [Security and Disclosure Information] - Guide for reporting vulnerabilities and the security release process
```

Comment:
Super tiny nit 😬 
but can you wrap this to conform with the 80 character limit used by the rest of the document and put a `.` on the end?